frontend/sozisel-app/src/components/PresenterSession/PresenterSession.tsx

```diff
@@ -56,16 +88,44 @@ export default function PresenterSession(): ReactElement {
             />
           )}
         </Grid>
-        <Grid item xs={6} className="firstRowItem">
-          <Paper elevation={2}></Paper>
-        </Grid>
-        <Grid item xs={3} className="firstRowItem">
+        {useJitsi && (
+          <Grid item xs={6} className="firstRowItem jitsiFrame">
+            <Paper elevation={2}>
+              {!jitsiLoading && jitsiData?.generateJitsiToken && (
+                <JitsiFrame
+                  roomId={id}
+                  token={jitsiData.generateJitsiToken.token}
+                  displayName={jitsiData.generateJitsiToken.displayName}
+                />
+              )}
+            </Paper>
+          </Grid>
+        )}
+
+        <Grid item xs={useJitsi ? 3 : 6} className="firstRowItem">
           {data && data.session && (
-            <ParticipantsList sessionId={data.session.id} />
+            <ParticipantsList
+              error={getParticipantsError}
+              loading={getParticipantsLoading}
+              participants={participants}
+            />
           )}
         </Grid>
         <Grid item xs={12} className="secondRowItem">
-          <Paper elevation={2}></Paper>
+          {data && data.session && (
+            <EventsTimeline
+              participants={participants.filter(
+                (participant) => participant.type !== "presenter"
+              )}
+              sessionId={data.session.id}
+              events={data.session.sessionTemplate.events}
+              launchedEvents={data.session.launchedEvents
```

Comment:
Do you want me to return already sorted entries on backend?

Reply:
ok, I will remove this sort.